invenio_damap/templates/semantic-ui/invenio_damap/macros/detail.html

```diff
@@ -9,10 +9,18 @@
 
 {%- macro render_dmp_button() %}
   {% set user_jwt = create_auth_jwt() %}
-  {% set dmp_list = query_damap_madmps(user_jwt)['hits']['hits'] if user_jwt is not none else [] %}
+  {% set dmp_list = query_damap_madmps(user_jwt) if user_jwt is not none else none %}
   {% set DAMAP_URL = get_damap_url() %}
 
-  <div class="column pt-5 pb-5" id="dmps-container" data-dmps='{{ dmp_list | tojson }}' data-damap-url="{{ DAMAP_URL }}">
-  </div>
-  {{ webpack['invenio-damap.js'] }}
+  {% if dmp_list is not none %}
```

Comment:
nit: for easier python readability
```suggestion
  {% if dmp_list is not None %}
```

Reply:
Same as above.